package.json

```diff
@@ -28,11 +28,14 @@
     "dive": "^0.5.0",
     "dotenv": "^16.0.3",
     "mongodb": "^5.1.0",
+    "node-fetch": "^2.6.11",
```

Comment:
in hindsight, can remove this since we already make use of of urllib (required for digest Auth fetch with Atlas Admin) to keep it light as possible but this is only few kB https://www.npmjs.com/package/node-fetch/v/2.6.11